pkg/apis/kai/v1/global.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2025 NVIDIA CORPORATION
+// SPDX-License-Identifier: Apache-2.0
+
+package v1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/utils/ptr"
+
+	"github.com/NVIDIA/KAI-scheduler/pkg/apis/kai/v1/common"
+)
+
+// GlobalConfig defines the global configuration of the system
+type GlobalConfig struct {
+	// Openshift configures the operator to install on Openshift
+	// +kubebuilder:validation:Optional
+	Openshift *bool `json:"openshift,omitempty"`
+
+	// Affinity defined affinity to the all microservices
+	// +kubebuilder:validation:Optional
+	Affinity *v1.Affinity `json:"affinity,omitempty"`
+
+	// SecurityContext defines security context for the kai containers
+	// +kubebuilder:validation:Optional
+	SecurityContext *v1.SecurityContext `json:"securityContext,omitempty"`
+
+	// ImagesPullSecret defines the container registry kai secret credential
```

Comment:
```suggestion
	// ImagesPullSecret defines an image pull secret for kai services, in cases where container registry credentials are required
```

Maybe phrasing could be improved even more,